src/app/shared/modules/sai-editor/sai-comment-generator.service.ts

```diff
@@ -247,6 +247,32 @@ export class SaiCommentGeneratorService {
           break;
       }
     }
+    	
+	if (actionLine.indexOf('_followGroupParamTwo_') > -1) {
+      switch (Number(smartScript.action_param2)) {
+        case 1:
+          actionLine = actionLine.replace('_followGroupParamTwo_', 'Circle');
+          break;
+        case 2:
+          actionLine = actionLine.replace('_followGroupParamTwo_', 'Semi-Circle Behind');
+          break;
+        case 3:
+          actionLine = actionLine.replace('_followGroupParamTwo_', 'Semi-Circle Front');
+          break;
+		case 4:
+          actionLine = actionLine.replace('_followGroupParamTwo_', 'Line');
+          break;
+		case 5:
+          actionLine = actionLine.replace('_followGroupParamTwo_', 'Column');
+          break;
+		case 6:
+          actionLine = actionLine.replace('_followGroupParamTwo_', 'Angular');
+          break;
+        default:
+          actionLine = actionLine.replace('_followGroupParamTwo_', '[Unknown Follow Type]');
+          break;
+      }
+    }
```

Comment:
you are using mixed spaces and tabs, I am cleaning it
moreover I am using a support variable to reduce the code redundancy

thanks for the PR btw :P, I am also adding some unit-tests to keep the code coverage at 100%

Reply:
I need to start using VStudio or VSCode lmao